constants/fonts.ts

```diff
@@ -11,6 +11,9 @@ export const headerMd = `${bold} 20px/28px ${family}`;
 export const headerSm = `${semiBold} 18px/25px ${family}`;
 export const headerXs = `${bold} 16px/22px ${family}`;
 
+export const textXXXl = `${regular} 96px/110px ${family}`;
```

Comment:
nit: seems like all of these bigger text styles are only used in heading-type places. Wouldn't it be easier to name the big ones like `headerXXL` rather than `textXXXL`?

Reply:
Sure, makes sense to me. Honestly, it would be nice if we condensed down some of these font styles, as it's getting bloated. It might be worth renaming some of the existing ones I guess.